mteb/models/e5_models.py

```diff
@@ -113,69 +108,27 @@
 ]
 
 
-class E5Wrapper:
-    """following the implementation within the Scandinavian Embedding Benchmark and the intfloat/multilingual-e5-small documentation."""
-
-    def __init__(
-        self,
-        model_name: str,
-        sep: str = " ",
-        prompt_name: str | None = None,
-        **kwargs: Any,
-    ):
-        self.model_name = model_name
-        self.mdl = SentenceTransformer(model_name)
-        self.sep = sep
-
-    def to(self, device: torch.device) -> None:
-        self.mdl.to(device)
-
-    def encode(  # type: ignore
-        self,
-        sentences: list[str],
-        *,
-        batch_size: int = 32,
-        **kwargs: Any,
-    ):
-        return self.encode_queries(sentences, batch_size=batch_size, **kwargs)
-
-    def encode_queries(
-        self,
-        queries: list[str],
-        batch_size: int = 32,
-        prompt_name: str | None = None,
-        **kwargs: Any,
-    ):
-        sentences = ["query: " + sentence for sentence in queries]
-        emb = self.mdl.encode(sentences, batch_size=batch_size, **kwargs)
-        return emb
-
-    def encode_corpus(
-        self,
-        corpus: list[dict[str, str]] | dict[str, list[str]],
-        prompt_name: str | None = None,
-        batch_size: int = 32,
-        **kwargs: Any,
-    ):
-        if "request_qid" in kwargs:
-            kwargs.pop("request_qid")
-        sentences = corpus_to_texts(corpus)
-        sentences = ["passage: " + sentence for sentence in sentences]
-        emb = self.mdl.encode(sentences, batch_size=batch_size, **kwargs)
-        return emb
-
-
 e5_mult_small = ModelMeta(
-    loader=partial(E5Wrapper, model_name="intfloat/multilingual-e5-small"),  # type: ignore
+    loader=partial(
+        sentence_transformers_loader,
+        model_name="intfloat/multilingual-e5-small",
+        revision="fd1525a9fd15316a2d503bf26ab031a61d056e98",
+        prompts={"query": "query: ", "passage": "passage: "},
+    ),
     name="intfloat/multilingual-e5-small",
     languages=XLMR_LANGUAGES,
     open_source=True,
-    revision="e4ce9877abf3edfe10b0d82785e83bdcb973e22e",
+    revision="fd1525a9fd15316a2d503bf26ab031a61d056e98",
```

Comment:
?

Reply:
For some reason, `e4ce987` no longer exists as a revision in the Hub: https://huggingface.co/intfloat/multilingual-e5-small/commits/main